exercise_3/src/Student.cpp

```diff
@@ -0,0 +1,22 @@
+#include <sstream>
+#include <string>
+#include "Student.h"
+
+Student::Student(int &id, std::string &name) {
+    this -> _id = id;
```

Comment:
1. "this->" is redundand.
2. You may initialize members on initialization list.
3. Make arguments const.